doc/manual/de/config/widgets/line/index.rst

```diff
@@ -10,8 +10,11 @@ Das Line Widget
 Beschreibung
 ------------
 
-Das Line Widget fügt der Visualisierung eine horizontale Linie hinzu. Hiermit kann man die Visu grafisch strukturieren.
-Farben und Stile werden durch das gewählte Design bestimmt und lassen sich nicht ändern.
+Das Line Widget fügt der Visualisierung eine dicke horizontale Linie hinzu. Hiermit kann man die Visu grafisch strukturieren.
+Farben und Stile werden durch das gewählte Design bestimmt und lassen sich nicht ändern. 
+
+Das Line Widget kann auch im Navbar verwendet werden. Im Top bzw. Bottom Navbar werden damit dünne senkrechte Trennlinien erzeugt, 
```

Comment:
... "in den Navbars"..."In der Top- bzw. Bottom-Navbar" klingt für mich irgendwie richtiger ?!